### PR TITLE
Horizon 32 Release Notes

### PR DESCRIPTION
**Pull request by RangerRick**
Note: I will make another push with the codename put in the `changelog.adoc` once voting has closed :)

### CODE REVIEW

File: docs/modules/releasenotes/pages/whatsnew.adoc [hunk 1]
```diff
@@ -7,7 +7,12 @@
...
 
-== New features and improvements
+== New features and important changes
+
+=== JDK 17
+
+Horizon 32 adds support for running on JDK 17, while retaining support for JDK 11.
+We are not (yet) taking advantage of JDK 17 specific features, but the new JDK provides a bunch of performance improvements under the covers that benefit any application running on it.
 
 === Newts: DataStax driver for Cassandra upgrade
```
Comment (mmahacek) [inline docs/modules/releasenotes/pages/whatsnew.adoc:17]: Line 26 appears to be self-referential. It mentions editing `newts-on-cassandra.conf` with a property pointing to itself.  Should that reference an `opennms.properties.d` file?

File: docs/modules/releasenotes/pages/whatsnew.adoc [hunk 2]
```diff
@@ -50,6 +55,40 @@ You can continue to use the filter as-is; no changes are needed to use this fore
 
 The filter that depends on the `R` package will be discontinued in a future release.
 
+=== Enhanced Linkd Improvements
```
Comment (mmahacek) [inline docs/modules/releasenotes/pages/whatsnew.adoc:58]:
```suggestion
=== Enhanced Linkd improvements
```

File: docs/modules/releasenotes/pages/whatsnew.adoc [hunk 3]
```diff
@@ -50,6 +55,40 @@ You can continue to use the filter as-is; no changes are needed to use this fore
...
+Enlinkd has been refactored, improving performance, fixing bugs, and adding a number of new features, most notably new Karaf command-line tools:
+
+* enlinkd-discovery-bridge-domain: triggers bridge topology discovery
+* enlinkd-run-collection: triggers an SNMP linkd collection
+* enlinkd-reload: restarts Enlinkd
+* enlinkd-reload-config: reload the Enlinkd configuration
+* enlinkd-reload-topology: refresh topology to force an update based on enlinkd data
+* enlinkd-snmp-collect: perform an Enlinkd SNMP group/table collection
+
+=== Poll Status History Enabled By Default
```
Comment (mmahacek) [inline docs/modules/releasenotes/pages/whatsnew.adoc:69]:
```suggestion
=== Poll status history enabled by default
```

File: docs/modules/releasenotes/pages/whatsnew.adoc [hunk 4]
```diff
@@ -50,6 +55,40 @@ You can continue to use the filter as-is; no changes are needed to use this fore
...
+
+Our service monitor infrastructure has had support for storing the up/down history of poll status, but it is now enabled by default, including graph definitions for all included services.
+This allows you to get historical availability per-service beyond 24 hours.
+
+=== Experimental Asynchronous Polling
```
Comment (mmahacek) [inline docs/modules/releasenotes/pages/whatsnew.adoc:74]:
```suggestion
=== Experimental asynchronous polling
```

File: docs/modules/releasenotes/pages/whatsnew.adoc [hunk 5]
```diff
@@ -50,6 +55,40 @@ You can continue to use the filter as-is; no changes are needed to use this fore
...
+
+The OpenNMS Plugin API has been updated to version 1.5.0, which includes additional alarm- and ticketer-related APIs for plugin authors.
+
+=== Usage statistics
+
+Consent to share collected usage statistics is now assumed by default on new installs.
+Existing installs respect the setting in effect before upgrading.
+
+For more details, see the xref:deployment:core/getting-started.adoc#usage-statistics[admin guide].
+
```
Comment (mmahacek) [inline docs/modules/releasenotes/pages/whatsnew.adoc:91]:
```suggestion

=== Additional self-monitoring

Additional JDBC collections for tracking database performance have been added to the built-in monitoring of the localhost node.

```

File: docs/modules/releasenotes/pages/whatsnew.adoc [hunk 6]
```diff
@@ -7,7 +7,12 @@
 * *Java 11 and 17*: OpenNMS Horizon 32 runs on JDKs 11 and 17.
 * *PostgreSQL 10 or higher*: Horizon 32 requires any supported version of PostgreSQL from 10 up to (and including) 15.
 
-== New features and improvements
+== New features and important changes
+
+=== JDK 17
+
+Horizon 32 adds support for running on JDK 17, while retaining support for JDK 11.
+We are not (yet) taking advantage of JDK 17 specific features, but the new JDK provides a bunch of performance improvements under the covers that benefit any application running on it.
```
Comment (mmahacek) [inline docs/modules/releasenotes/pages/whatsnew.adoc:15]:
```suggestion
We are not (yet) taking advantage of JDK 17 specific features, but the new JDK provides a bunch of performance improvements under the covers that benefit any application running on it.

=== Kafka version 3.x

Documentation has been updated to indicate that Kafka version 3 is supported for the message broker.
```

File: docs/modules/releasenotes/pages/whatsnew.adoc (hunk 1 above)
Comment (RangerRick) [reply docs/modules/releasenotes/pages/whatsnew.adoc:17]: yeah, probably... @j-white wrote that -- pretty sure that should be `opennms.properties.d`

File: docs/modules/releasenotes/pages/whatsnew.adoc (hunk 4 above)
Comment (RangerRick) [reply docs/modules/releasenotes/pages/whatsnew.adoc:78]: these always throw me off... they're "headers" so my brain assumes they should be Capitalized, hehe

**Review by mershad-manesh — APPROVED** [review]
LGTM

**Comment by RangerRick** [comment]
@mmahacek @Bonrob2 feel free to ping if you think of anything, but I'm gonna go ahead and merge this so @mershad-manesh can get crackin'

**Pull request merged**